BuildAndTest.cmd

```diff
@@ -63,6 +63,11 @@ call :CreatePublishPackage net6.0 win-x64 || goto :ExitFailed
 call :CreatePublishPackage net6.0 linux-x64 || goto :ExitFailed
 call :CreatePublishPackage net6.0 osx-x64 || goto :ExitFailed
 
+::Create the BinSkim dotnet tool publish packages
+echo Creating BinSkim dotnet tool publish packages
+dotnet publish %~dp0src\BinSkim.Driver\BinSkim.Driver.csproj --no-restore -c %Configuration% -f net6.0 -p:PublishDir=%~dp0bld\bin\%Platform%_%Configuration%\DotNetToolPublish\net6.0\any
```

Comment:
>dotnet [](http://example.com/codeflow?start=0&length=6)

this can be done in `:CreatePublishPackage` by adding `PublishDir` as a new parameter

Reply:
The reason why :CreatePublishPackage is not used is that there is no runtimeID (win-64/linux-64/osx-64) to provide. The files are published into %~dp0bld\bin\%Platform%_%Configuration%\DotNetToolPublish\net6.0\**any** folder